GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -959,7 +959,7 @@ private void InternalInitialize()
 
                     _formBrowseMenus.InsertRevisionGridMainMenuItems(repositoryToolStripMenuItem);
 
-                    // Request all branches if side panel is shown
+                    // Request all branches if leftpanel is shown
```

Comment:
```suggestion
                    // Request all branches if left panel is shown
```

Ditto other places

Reply:
That was intentional, to use leftpanel consistently as the name, not "left panel" "leftpanel" and "left-panel" used randomly.
But it can be changed.